src/ext/VSExtension/wixlib/VsixPackage.wxs

```diff
@@ -22,13 +22,19 @@
             </RegistrySearch>
         </Property>
 
-        <!-- VS2015 is the latest VSIX at this point in time, so search for that by default. -->
         <Property Id="VS2015_VSIX_INSTALLER_PATH" Secure="yes">
             <RegistrySearch Id="WixVS2015EnvDir" Root="HKLM" Key="SOFTWARE\Microsoft\VisualStudio\14.0\Setup\VS" Name="EnvironmentDirectory" Type="directory">
                 <FileSearch Id="WixVS2015VsixInstaller" Name="VSIXInstaller.exe" />
             </RegistrySearch>
         </Property>
 
+        <!-- Currently, VS2017 contains the latest VSIX installer, so search it by default. -->
+        <Property Id="VS2017_VSIX_INSTALLER_PATH" Secure="yes">
+            <DirectorySearchRef Id="VS2017EnvironmentDirectorySearch" Parent="VS2017DirectorySearch" Path="Common7\IDE">
+                <FileSearch Id="WixVS2017VsixInstallerPath" Name="VSIXInstaller.exe" />
```

Comment:
Would you kindly elaborate what is that search engine root and there it is defined?
I suppose it's %ProgramFiles(x86)%\Microsoft Visual Studio\Installer but I haven't seen this path being used in wix3 sources.